source/upgrade.txt

```diff
@@ -76,6 +76,16 @@ Version 6.x Breaking Changes
 - Automatic Encryption methods are now in the {+driver-short+}. You must
   import these methods from the driver instead of from
   ``mongodb-client-encryption``.
+- Removed the ``ObjectId`` constructor that accepted a 12-character string.
+- The ``abortTransaction()`` and ``commitTransaction()`` methods return
+  ``null`` instead of the raw command results.
+- Removed connection option helpers that accepted values other than ``true``
+  or  ``false`` as booleans. You must provide either ``true`` or ``false`` values in
+  the connection string or to the MongoClient constructor.
+- Removed the Binary BSON type constructor that accepted a string.
```

Comment:
Suggestion:
I think the monospace formatting for "Binary" should still apply since it is a type/class name and helps the reader understand which type (in case there are multiple binary types).